api/tests/hours_test.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from api.models import Hour, Location
+
+
+class HourTest(TestCase):
+    sample_location_data = {'name': 'Youth Green',
+                            'category': 'Community Garden',
+                            'address_1': '12467 W. Osborne Street',
+                            'city': 'Los Angeles',
+                            'state': 'CA',
+                            'zipcode': '91331',
+                            'latitude': 34.25777331,
+                            'longitude': -118.4046685}
+    sample_hour_data = {'day': 'AllMo',
+                        'open_time': '08:00',
+                        'close_time': '14:00',
+                        }
+
+    def test_creating_hour(self):
+        loc = Location(**self.sample_location_data)
+        loc.save()
+        self.sample_hour_data['location_id'] = loc.id
+        hour = Hour(**self.sample_hour_data)
```

Comment:
Small nitpick here but I think i would rather set the location ID on the hour object after we create it rather than assigning it into the `sample_hour_data` map, for the reason that assigning it into the map will carry over onto later tests where that location ID might not be valid any more. (at this point we only have the one test so it wouldn't really affect anything but I could see it potentially becoming confusing whenever someone comes back in here to add another test)

Also if i was going to the trouble to assign the location_id like that i would probably go ahead and throw in a test for the association (verifying that that hour's location matches the one created the line before). otherwise I think someone reading the test is a little confused why we are taking the trouble to create the location and assign the ID anyway.

my thought is something like this:

```python
    def test_creating_hour(self):
        loc = Location(**self.sample_location_data)
        loc.save()

        hour = Hour(**self.sample_hour_data)
        hour.location_id = loc.id
        hour.save()

        query = Hour.objects.filter(day='AllMo')
        hour = query[0]

        self.assertEqual(1, len(query))
        self.assertEqual('AllMo', hour.day)
        self.assertEqual('08:00:00', str(hour.open_time))
        self.assertEqual('14:00:00', str(hour.close_time))
        self.assertEqual(loc, hour.location)

```